client/src/components/ManageArtist/ManageArtistContainer.tsx

```diff
@@ -3,42 +3,43 @@ import React from "react";
 import { bp } from "../../constants";
 import { Outlet } from "react-router-dom";
 import ArtistHeaderSection from "../common/ArtistHeaderSection";
+import { useGlobalStateContext } from "state/GlobalState";
 import { useTranslation } from "react-i18next";
 import styled from "@emotion/styled";
 import { useArtistContext } from "state/ArtistContext";
 
-const Container = styled.div<{ artistBanner: boolean }>`
-
-
+const Container = styled.div<{ artistBanner: boolean; userId?: number }>`
   width: 100%;
-
-  @media screen and (max-width: ${bp.medium}px) {
-    padding: var(--mi-side-paddings-xsmall);
-  }
-
   margin-top: calc(16vh);
 
   ${(props) => (!props.artistBanner ? "margin-top: 0px;" : "")}
   max-width: calc(1080px + 4rem);
 
   @media screen and (max-width: ${bp.medium}px) {
+    padding: var(--mi-side-paddings-xsmall);
     padding: 0rem !important;
     width: 100%;
-    // margin-top: 55px;
-    // margin-top: 30vw;
-    margin-top: 0;
+    ${(props) => (props.artistBanner ? "margin-top: 0px;" : "")}
+    ${(props) => (!props.userId ? "margin-top: 13vh;" : "")}
+    ${(props) => (!props.artistBanner ? "margin-top: 0px;" : "")}
```

Comment:
The `?` indicates a [ternary clause](https://stackoverflow.com/questions/392932/how-do-i-use-the-conditional-ternary-operator). 

```
condition ? do-the-true-thing : do-the-false-thing
```

So having a function call for both `!props.artistBanner` and `props.artistBanner` is redundant. You can instead do: 

```suggestion
    ${(props) => (props.artistBanner ? "margin-top: 0px;" : "margin-top: 0px;")}
    ${(props) => (!props.userId ? "margin-top: 13vh;" : "")}
```

Which is telling me that basically you're saying "if there is an artistBannder, do 0px margin, if there isn't an artistBanner, do 0px margin". Seems redundant? You should just have margin-top: 0px;

```suggestion
    margin-top: 0px;
    ${(props) => (!props.userId ? "margin-top: 13vh;" : "")}
```